src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -578,4 +580,37 @@ public static void fillNormal(int[] faceData, EnumFacing facing)
         if(part.isPresent()) return Optional.absent();
         return Optional.of(new TRSRTransformation(matrix));
     }
+
+    private static int glintValue;
+
+    public static boolean renderItemGlint(ItemStack stack, IBakedModel model)
+    {
+        OverlayGlintEvent.StackOverlayEvent event = new OverlayGlintEvent.StackOverlayEvent(stack, model);
+        if(!MinecraftForge.EVENT_BUS.post(event))
+        {
+            glintValue = (event.glintValue);
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean renderArmorGlint(ItemStack stack, ModelBase model, EntityLivingBase entity, int slot)
+    {
+        OverlayGlintEvent.ArmorOverlayEvent event = new OverlayGlintEvent.ArmorOverlayEvent(stack, model, entity, slot);
+        if(!MinecraftForge.EVENT_BUS.post(event))
+        {
+            glintValue = event.glintValue;
+            return true;
```

Comment:
Passing in the itemstack would prevent the need for this potentially thread-unsafe static variable here.